vndr.cfg

```diff
@@ -0,0 +1,11 @@
+github.com/Sirupsen/logrus 26709e2714106fb8ad40b773b711ebce25b78914
```

Comment:
This could be `v0.8.0` or probably even `v0.10.0`. :smile: (https://github.com/Sirupsen/logrus/commit/26709e2714106fb8ad40b773b711ebce25b78914)